src/components/techs/TechItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
I see you destructure `tech` on line 4 but you don't destructure `object` on line 2.
Is there a reason?

Reply:
Because the "tech" is passed to PropTypes or I don't have any other explanation.